src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -20,273 +20,980 @@ PICS:
 
 config:
     nodeId: 0x12344321
-    cluster: "Basic Information"
-    endpoint: 0
+    cluster: "Scenes"
+    endpoint: 1
 
 tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
     - label: "TH sends a RemoveAllGroups command to DUT."
       PICS: G.S.C04.Rsp
-      verification: |
-          If a status response is expected, DUT sends a response to TH with the Status field equal to 0x00 (SUCCESS).
-      disabled: true
-
-    - label:
-          "preparation step for using commands from Groups cluster: TH generates
-          fabric-unique GroupID, GroupName, random key, EpochKey0 and
-          GroupKeySetID. TH sets GroupKeySecurityPolicy = TrustFirst (1)
-          GroupKeyMulticastPolicy = PerGroupID (0) EpochStartTime0 = 0"
-      verification: |
-
-      disabled: true
+      cluster: "Groups"
+      endpoint: "1"
+      command: "RemoveAllGroups"
+
+    - label: "TH reads from the DUT the (0x0006) SceneTableSize attribute"
+      PICS: S.S.A0006
+      command: "readAttribute"
+      attribute: "SceneTableSize"
+      response:
+          values:
+              - name: "SceneTableSize"
+                saveAs: maxScenes
+          constraints:
+              type: uint16
+
+    - label:
+          "preparation step for using commands from Groups cluster: Add KeySet"
+      cluster: "Group Key Management"
+      endpoint: 0
+      command: "KeySetWrite"
+      arguments:
+          values:
+              - name: "GroupKeySet"
+                value:
+                    {
+                        GroupKeySetID: 0x01a1,
+                        GroupKeySecurityPolicy: 1,
```

Comment:
Again, this is going to fail...

Reply:
Created [issue ](https://github.com/CHIP-Specifications/chip-test-plans/issues/2746) and changed to 0.